xbmc/video/videosync/VideoSyncCocoa.cpp

```diff
@@ -142,12 +125,19 @@ void CVideoSyncCocoa::OnResetDevice()
 
 float CVideoSyncCocoa::GetFps()
 {
+  int fpsInt;
 #if defined(TARGET_DARWIN_IOS)
-  UpdateFPS(g_Windowing.GetDisplayLinkFPS() + 0.5);
+  fpsInt = MathUtils::round_int(g_Windowing.GetDisplayLinkFPS() + 0.5);
```

Comment:
try to drop it then you should see proper values like 23.976 as fps on the codec screen ('o').